### ISSUE TITLE
[translations] marking English not to translate

### ISSUE THREAD

**Issue by shawna-slh** [w3c/wai-translations#102]
We say to translate the words but not the acronym for WCAG and such
 https://github.com/w3c/translation-glossaries/blob/main/general.md
e.g., Richtlinien für barrierefreie Webinhalte (WCAG)

afaik, Things that should not be translated should be marked up:
* lang="en"
* translate="no" (see https://www.w3.org/International/questions/qa-translate-flag.en)

To Do:
* [ ] research best way to do this
* [ ] add to translation instructions
* [ ] update all English pages that have such
* [ ] update all translations with it
phew!
